### DISCUSSION TITLE
Mantine UI and Chakra UI status/ETA

### DISCUSSION THREAD

**Discussion by ockam** [refinedev/refine#2409] [Q&A] [answered]
Hi,

the docs mention upcoming support for Mantine UI, but I also saw a branch for Chakra UI in the repo.

Can you give us a status on your progress for both?

I’d really like to see Chakra UI support lands in Refine. 🤞🏼

**Comment by omeraplak** [comment, marked as answer]
Hey @ockam 👋,
Thank you for contacting us! We will be releasing both our Mantine and Chakra UI support by mid-October. Documents may take some time because we will update our document structure. But don't worry, you won't need much documentation because we'll publish enough examples 🚀

**Reply by ockam** [reply]
Thanks for the update!

Don’t mean to put any pressure, but we’re about to kickstart a new project; do you think we could work from the Chakra branch right away (and beta test it)?

**Reply by omeraplak** [reply]
Chakra UI branch was created in our 1-day internal hackathon. So it's not usable, but it can inspire you🚀

**Reply by omeraplak** [reply]
Hey @ockam ,
We've released `@pankod/refine-mantine` 💥 We will publish our tutorial soon now you can consume examples 🍕

Examples;
https://refine.dev/docs/examples/form/mantine/useDrawerForm/
Form Docs;
https://refine.dev/docs/api-reference/mantine/hooks/form/useForm/

**Reply by rvlasenko** [reply]
@omeraplak Hi, thanks for including mantine! This UI framework is growing very fast, is there any way to update it to the latest version 5.7.0? I see @pankod/refine-mantine uses @mantine/core 5.5.6

**Reply by omeraplak** [reply]
Hey @ockam ,
Thank you for your kindly words. You can use it temporarily by installing 5.5.6 in your Project. We will update it with hotfix on monday 🚀

**Reply by omeraplak** [reply]
Hey @rvlasenko ,
We've fixed with [`@pankod/refine-mantine@1.11.4`](https://github.com/refinedev/refine/releases/tag/%40pankod%2Frefine-mantine%401.11.4). Could you approve this fix?

**Reply by rvlasenko** [reply]
@omeraplak Thanks for the update, the latest version is now using
